opm/parser/eclipse/RawDeck/StarToken.cpp

```diff
@@ -69,55 +71,38 @@ namespace Opm {
 
     template<>
     int readValueToken< int >( string_view view ) {
+        int n = 0;
```

Comment:
Sure - the main point was that as an old C hand I was expecting a modifiable scalar argument to be passed as `&n`; but I guess the signature of the parse routine is `(....,  T& value )`.